app/src/main/java/com/aleksandra/go4mytrip/PackingListRepository.kt

```diff
@@ -0,0 +1,21 @@
+package com.aleksandra.go4mytrip
+
+import com.google.firebase.auth.FirebaseAuth
+import com.google.firebase.database.DatabaseReference
+import com.google.firebase.database.FirebaseDatabase
+
+open class PackingListRepository {
+
+    private lateinit var uid: String
```

Comment:
if FirebaseAuth.getInstance().currentUser is null, then your application will crash